terratag_test.go

```diff
@@ -68,7 +68,12 @@ func TestTerraform1o1WithFilter(t *testing.T) {
 }
 
 func testTerraform(t *testing.T, version string) {
-	for _, tt := range getEntries(version) {
+	entries := getEntries(version)
```

Comment:
@RLRabinowitz - please let me know if you have any additional suggestions on preventing the issue.
One problem is that if golang does not find a test, it still returns success. So not sure if there is anything quick/easy to do about that.

Reply:
Hey @TomerHeber maybe I'm missing something here. Why is this necessary? I'm not seeing any issue on the matter

Reply:
Never mind, I've found [the issue](https://github.com/env0/terratag/issues/87), I understand that this PR solves one part of it, correct?

Reply:
@RLRabinowitz 

Yes. If the test exists but the folder is empty - it will fail.
If the test does not exist - it will succeed (this is just how golang behaves when running `go test -run ...`).

Not ideal. Happy for suggestions.